ONSdigital/eq-survey-runner
# Eq 1 questionnaire manager

**_What**_

Implements the QuestionnaireManager class and the interaction between main components needed to render a questionnaire and receive the users submission back.  Uses a stub Validator object but stores responses and validation results in the session.  Provides base empty implementations for all other components not currently needed for eq-1.

**_How to test**_

1) Check out the branch
2) Run the application
3) Login and view the questionnaire at `http://localhost:5000/questionnaire/1`
4) Enter some data and click one of the submit buttons.
5) Verify the post redirects you to the same page as a get request

**_Who can test**_

Anybody except @warrenbailey or @weapdiv-david, although probably @LJBabbage should look at this

**_Future Work**_

Python documentation and additional unit tests to follow in separate PR.  Integration with @LJBabbage work on validation
